pkg/cmd/roachtest/sqlalchemy.go

```diff
@@ -0,0 +1,359 @@
+// Copyright 2018 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"regexp"
+	"sort"
+	"strings"
+	"sync"
+	"unicode"
+)
+
+var sqlAlchemyResultRegex = regexp.MustCompile(`(?P<name>.*::.*::.*) (?P<result>PASSED|ERROR|FAILED)`)
+
+const sqlalchemyTestsest = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890"
+
+var knownTestCrashes = []string{
```

Comment:
`sqlAlchemyKnownTestCrashes` would be safer since this file is in a package with many other kinds of tests.